dockers/docker-orchagent/orchagent.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 EXIT_SWSS_VARS_FILE_NOT_FOUND=1
-SWSS_VARS_FILE=/usr/share/sonic/templates/swss_vars.j2
+SWSS_VARS_FILE=/etc/sonic/swss_syncd_vars.j2
```

Comment:
>/etc/sonic/swss_syncd_vars.j2 [](start = 15, length = 29)

Does it make more sense to keep it in original directory? User are not expected to modify this file.
BTW, keep the original filename still make sense to me.

Reply:
The original directory `/usr/share/sonic/templates` is only available in swss docker. The reason to move it to `\etc\sonic` is to make the same template file accessible to both swss and syncd. Also, as this template is going to be accessed by syncd as well, I renamed to reflect this.